packages/botonic-react/src/webchat/header.jsx

```diff
@@ -26,32 +27,47 @@ const Diffuse = styled(Flex)`
   border-radius: 6px 6px 0px 0px;
 `
 
-export const DefaultHeader = props => (
-  <Diffuse color={props.color}>
-    <Flex width={1 / 4}>
-      <img
-        style={{
-          height: 24,
-          paddingTop: 15,
-          paddingLeft: 15
-        }}
-        src={Logo}
-      />
-    </Flex>
-    <Flex width={1} flexDirection='column'>
-      <HeaderTitle>Botonic</HeaderTitle>
-      <Subtitle>Online</Subtitle>
-    </Flex>
-  </Diffuse>
-)
+export const DefaultHeader = props => {
+  let HeaderImage = props.webchatState.theme.headerImage
+  let headerTitle = props.webchatState.theme.headerTitle
+  headerTitle ? headerTitle : (headerTitle = 'Botonic')
+  let headerSubtitle = props.webchatState.theme.headerSubtitle
+  return (
+    <Diffuse color={props.color}>
+      <Flex width={1 / 4}>
+        {HeaderImage ? (
+          <HeaderImage />
+        ) : (
+          <img
+            style={{
+              height: 24,
+              paddingTop: 15,
+              paddingLeft: 15
+            }}
+            src={Logo}
+          />
+        )}
+      </Flex>
+      <Flex width={1} flexDirection='column'>
+        <HeaderTitle headerSubtitle={headerSubtitle}>{headerTitle}</HeaderTitle>
+        <Subtitle>{headerSubtitle}</Subtitle>
```

Comment:
is it ok that headerSubtitle is passed to both the HeaderTitle and the Subtitle?

Reply:
headerSubtitle is passed because we need to know if there are or not for the position of the title. 
margin-top: ${props => (props.headerSubtitle == undefined ? '18px' : 5)};

Reply:
This is not the best way to do this, fixed margins are hard to maintain. Usually you want to center vertically the content of the header, so it doesn't matter if you have 1, 2 or more elements, they'll always be displayed well. Then you add a "min-width" to the container element so that its height does not depend on the content.